source/jormungandr/tests/isochrone_tests.py

```diff
@@ -32,37 +32,52 @@
 from check_utils import *
 
 
-@dataset(["main_routing_test"])
+@dataset(["basic_routing_test"])
```

Comment:
Nice some tests :v: 

can you rename the class ? it should be called TestIsochrone no ?